### PR TITLE
Add OpenAI Gym Example

### PR DESCRIPTION
**Pull request by lukicdarkoo**
This PR adds a very simple reinforcement learning environment based on OpenAI Gym.

**Documentation**
https://cyberbotics.com/doc/guide/samples-howto?version=feature-opengym-example#openai_gym-wbt

### CODE REVIEW

File: docs/guide/samples-howto.md [hunk 1]
```diff
@@ -115,6 +115,15 @@ An alternative non blocking controller called `mouse_events_non_blocking.c` is a
 In this example, the omnidirectional wheels are modeled with two layers of joints and cylinders solids.
 Faster omnidirectional wheels implementations could be achieved using asymmetric friction (cf. `Youbot` model).
 
+
+### [openai\_gym.wbt]({{ url.github_tree }}/projects/samples/howto/openai_gym/worlds/openai_gym.wbt)
+
+**Keywords**: Reinforcement learning, OpenAI Gym
+
+![openai_gym.png](images/samples/openai_gym.thumbnail.jpg) This examples shows an integration of [OpenAI Gym](https://gym.openai.com/) with Webots.
```
Comment (omichel) [inline docs/guide/samples-howto.md:123]:
```suggestion
![openai_gym.png](images/samples/openai_gym.thumbnail.jpg) This example shows an integration of [OpenAI Gym](https://gym.openai.com/) with Webots.
```

File: projects/samples/howto/openai_gym/worlds/openai_gym.wbt [hunk 2]
```diff
@@ -0,0 +1,73 @@
+#VRML_SIM R2021a utf8
+WorldInfo {
+  coordinateSystem "NUE"
+    info [
+    "This examples shows an integration of OpenAI Gym with Webots."
```
Comment (omichel) [inline projects/samples/howto/openai_gym/worlds/openai_gym.wbt:5]:
```suggestion
    "This example shows an integration of OpenAI Gym with Webots."
```

**Review by omichel — CHANGES_REQUESTED** [review]
I believe this is great contribution.
We should add a line in the change log about it.
I am also wondering why calling the controller "openai_gym_environment" and why not simply "openai_gym"?

**Comment by lukicdarkoo** [comment]
> I am also wondering why calling the controller "openai_gym_environment" and why not simply "openai_gym"?

No good reason. In OpenAI they call it "environment", so I wanted to be close to their naming scheme.

**Comment by omichel** [comment]
> No good reason. In OpenAI they call it "environment", so I wanted to be close to their naming scheme.

I see. Maybe, I would simplify to "openai_gym" as it looks simpler.

**Review by omichel — APPROVED** [review]
Thank you.

**Pull request merged**